### DISCUSSION TITLE
听说APP端不能发帖？

### DISCUSSION THREAD

**Discussion by fuyutukikuze** [share121/inter-knot#4788] [其他]
让我试试发吧！
![20240818150508](https://github.com/user-attachments/assets/34aaeb4c-1d07-4faf-8d47-0cd149e07036)

**Comment by nzj2004** [comment]
可以的，我昨天试过

**Reply by fuyutukikuze** [reply]
目前这篇帖子就是APP端发的。要是界面能改简中就好了。😭

**Reply by duoduo13123678** [reply]
除非GitHub给你上简中，另外好像是我提的在APP里发帖的功能（或者别人提的相同意见）

**Reply by cuitaoyi** [reply]
我在app点击评论就自动跳到浏览器来了，我在app发不出贴😂

**Comment by JT9364** [comment]
绳网改版后就不知道怎么发帖了（笑

**Reply by konghuan7759** [reply]
一样，APP端的功能键好像找不到了

**Reply by JT9364** [reply]
找到了，只需要在对应浏览器修改为《桌面版网站》app右下角就会有编写图标了（貌似只有电脑版才能编写帖子，果然这个思路才是正确的）

**Comment by PB4437** [comment]
为什么我要发帖的，但是在选择类型那里没有按键

**Comment by BSJ3456A** [comment]
我点了编辑按钮 然后给我跳转到这来了 也没有发帖选项 只能看贴 ꒦ິ^꒦ິ

**Comment by cpc1212** [comment]
让我来傻乎乎的测试一下

**Comment by liang-TEA** [comment]
让我来试试回复一下
